modules/jip/functions/JIP/fn_JIP_ClientInit.sqf

```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+
+#include "..\..\settings.sqf"
+
+if (hasInterface && {(str (side player)) != "LOGIC"}) then {
```

Comment:
can use an exitwith instead. !hasInterface will detect server + clients. `player` wont exist on the headless client/server.

Reply:
Fixed.